app/scripts/HorizontalChromosomeLabels.js

```diff
@@ -61,6 +62,18 @@ class HorizontalChromosomeLabels extends PixiTrack {
     };
     this.stroke = colorToHex(this.pixiTextConfig.stroke);
 
+    // text objects to use if the tick style is "bounds", meaning
+    // we only draw two ticks on the left and the right of the screen
+    this.gBoundTicks = new PIXI.Graphics();
+
+    this.leftBoundTick = new PIXI.Text('', this.pixiTextConfig);
+    this.rightBoundTick = new PIXI.Text('', this.pixiTextConfig);
+
+    this.gBoundTicks.addChild(this.leftBoundTick);
+    this.gBoundTicks.addChild(this.rightBoundTick);
+
+    this.pMain.addChild(this.gBoundTicks);
```

Comment:
I wonder if it makes sense to wrap this into a function called `initBoundTicks()` together with a function `removeBoundTicks()` and only create these graphics objects when needed.

It might seem like overkill but I started noticing that we're already have an enormous amount of graphics objects in our scenegraph and it's starting to slow down things when there are a lot of tracks. Since `this.gBoundTicks` is essentially unused unless someone sets `tickPosition == 'ends'` I think we shouldn't create them unless needed. Or maybe we can reuse the same graphics object for even and end ticks?

Reply:
Makes sense, I'll update the PR 👍